yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/kernelSDK/KernelSDKIntentService.java

```diff
@@ -38,6 +36,7 @@
     private static final String ACTION_BACKEND_CALL = "com.yoti.mobile.android.sdk.network.action.BACKEND_CALL";
     private static final String ACTION_START_SCENARIO = "com.yoti.mobile.android.sdk.network.action.START_SCENARIO";
     private static final String YOTI_CALLED_RESULT_RECEIVER = "com.yoti.mobile.android.sdk.network.action.YOTI_CALLED_RESULT_RECEIVER";
+    public static final String YOTI_PENDING_INTENT_EXTRA = "YOTI_PENDING_INTENT_EXTRA";
```

Comment:
why not being consistent with the reverse dns notation like it has been used above?